src/dmd/typesem.d

```diff
@@ -795,15 +797,15 @@ private extern (C++) final class TypeSemanticVisitor : Visitor
             errors |= tf.checkRetType(loc);
             if (tf.next.isscope() && !(sc.flags & SCOPEctor))
             {
-                mtype.error(loc, "functions cannot return scope %s", tf.next.toChars());
+                mtype.error(loc, "functions cannot return scope `%s`", tf.next.toChars());
```

Comment:
Should the tick marks surround "return scope" also? "functions cannot \`return scope %s\`".  I'm not sure.

Reply:
Again, this is just a first approximation. There are thousands of messages to fix. The last time I tried to do this, the PRs all got derailed into debates about such things, and the result was zero progress for the following 6 months.

Reply:
> There are thousands of messages to fix. 

Are you sure you don't want to automate this?

Reply:
Others have said they were going to automate it. I'm not waiting any longer.